coffeescript from cdnjs is unusable

https://github.com/jashkenas/coffeescript/issues/4001


<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/24046014-coffeescript-from-cdnjs-is-unusable?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github).
</bountysource-plugin>

@PeterDaveHello -- Here's a dump of cdnjs coffee-script versions and working status:

version | size | output | status
--------- | ---------- | ------ | -------
**1.1.2** | *172K* | **CoffeeScript.VERSION == 1.1.2** | :coffee:
**1.1.3** | *16K* | *Error: Cannot find module './lexer'* | :paw_prints:
**1.2.0** | *176K* | **CoffeeScript.VERSION == 1.2.0** | :coffee:
**1.3.0** | *16K* | *Error: Cannot find module './lexer'* | :paw_prints:
**1.3.1** | *180K* | **CoffeeScript.VERSION == 1.3.1** | :coffee:
**1.3.2** | *16K* | *Error: Cannot find module './lexer'* | :paw_prints:
**1.3.3** | *184K* | **CoffeeScript.VERSION == 1.3.3** | :coffee:
**1.4.0** | *184K* | **CoffeeScript.VERSION == 1.4.0** | :coffee:
**1.5.0** | *16K* | *Error: Cannot find module './lexer'* | :paw_prints:
**1.6.0** | *16K* | *Error: Cannot find module './lexer'* | :paw_prints:
**1.6.1** | *16K* | *Error: Cannot find module './lexer'* | :paw_prints:
**1.6.2** | *204K* | **CoffeeScript.VERSION == 1.6.2** | :coffee:
**1.6.3** | *204K* | **CoffeeScript.VERSION == 1.6.3** | :coffee:
**1.7.0** | *24K* | *Error: Cannot find module './lexer'* | :paw_prints:
**1.7.1** | *212K* | **CoffeeScript.VERSION == 1.7.1** | :coffee:
**1.8.0** | *24K* | *Error: Cannot find module './lexer'* | :paw_prints:
**1.9.0** | *24K* | *Error: Cannot find module './lexer'* | :paw_prints:
**1.9.1** | *24K* | *Error: Cannot find module './lexer'* | :paw_prints:
**1.9.2** | *24K* | *Error: Cannot find module './lexer'* | :paw_prints:
**1.9.3** | *24K* | *Error: Cannot find module './lexer'* | :paw_prints:

note: the above table markdown can be reproduced using:
```sh
[cdnjs]# for version in ./ajax/libs/coffee-script/*/ ; do printf "**%s** | *%s* | %s\n" $(basename $version) $(du -h $version | cut -f1) "$(node -e "try { console.log('**CoffeeScript.VERSION == %s** | :coffee:', require('$version/coffee-script.min').CoffeeScript.VERSION) } catch(e) { console.log('*'+e+'* | :paw_prints:') }" )" ; done
```

Also since file size currently works as way to detect status, a report that groups non-working/working versions together is roughly:
```sh
[cdnjs]# du -hs ajax/libs/coffee-script/*/ | sort -n
16K     ajax/libs/coffee-script/1.1.3/
16K     ajax/libs/coffee-script/1.3.0/
16K     ajax/libs/coffee-script/1.3.2/
16K     ajax/libs/coffee-script/1.5.0/
16K     ajax/libs/coffee-script/1.6.0/
16K     ajax/libs/coffee-script/1.6.1/
24K     ajax/libs/coffee-script/1.7.0/
24K     ajax/libs/coffee-script/1.8.0/
24K     ajax/libs/coffee-script/1.9.0/
24K     ajax/libs/coffee-script/1.9.1/
24K     ajax/libs/coffee-script/1.9.2/
24K     ajax/libs/coffee-script/1.9.3/
172K    ajax/libs/coffee-script/1.1.2/
176K    ajax/libs/coffee-script/1.2.0/
180K    ajax/libs/coffee-script/1.3.1/
184K    ajax/libs/coffee-script/1.3.3/
184K    ajax/libs/coffee-script/1.4.0/
204K    ajax/libs/coffee-script/1.6.2/
204K    ajax/libs/coffee-script/1.6.3/
212K    ajax/libs/coffee-script/1.7.1/
```

--- a/ajax/libs/coffee-script/package.json
+++ b/ajax/libs/coffee-script/package.json
@@ -1,6 +1,6 @@
 {
   "name": "coffee-script",
-  "filename": "coffee-script.min.js",
+  "filename": "coffeescript.js",
   "version": "1.12.8",
   "description": "Unfancy JavaScript",
   "homepage": "http://jashkenas.github.com/coffee-script/",
@@ -12,12 +12,12 @@
     "script",
     "popular"
   ],
-  "npmName": "coffee-script",
+  "npmName": "coffeescript",
   "npmFileMap": [
     {
-      "basePath": "lib/coffee-script",
+      "basePath": "lib/coffeescript-browser-compiler-modern",
       "files": [
-        "coffee-script.js"
+        "coffeescript.js"
       ]
     }
   ],

